tasks/main.yml

```diff
@@ -0,0 +1,46 @@
+---
+- name: Calculate facts
+  set_fact:
+    build_base: "{{ (mumble_web.build_base | default(mumble_web_build_base)) | regex_replace('\\/$', '') }}"
```

Comment:
Why the regex? I think paths will be sanitized by ansible modules

Reply:
The problem was if the path has a trailing slash, the filter `dirname` will output the full path. For `build_base` it is actually not needed, but for `deploy_path` it is and I'd rather keep them similar.

Other possibility: Expect a path without trailing slash and document that.